src/components/MultiGesture.tsx

```diff
@@ -87,6 +88,9 @@ class MultiGesture extends React.Component<MultiGestureProps> {
 
       onPanResponderMove: (e: GestureResponderEvent, gestureState: GestureState) => {
         if (this.abandon) {
+          if (this.props.onCancel) {
+            this.props.onCancel()
+          }
```

Comment:
You can also use the fancy new [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) operator: `this.props.onCancel?.()`